docs/installation.rst

```diff
@@ -173,21 +173,23 @@ as single required argument.
 
 The script shows you some important information for the next steps:
 
-- the path to the main configuration file like
-  ``/etc/temboard-agent/11/main/temboard-agent.conf``.
-- TCP port like 2345.
+- the path to the main agent configuration file like
+  ``/etc/temboard-agent/11/main/temboard-agent.conf`` (note that some parts of the
```

Comment:
La note est pertinente, peux-tu la mettre en note avec:

``` rst
.. note:: Some parts...
```

Tu as un exemple dans le même fichier.

Reply:
done and pushed